sdk-actors/src/main/java/io/dapr/actors/client/DaprGrpcClient.java

```diff
@@ -38,6 +42,15 @@
    */
   @Override
   public Mono<byte[]> invoke(String actorType, String actorId, String methodName, byte[] jsonPayload) {
+    return invoke(actorType, actorId, methodName, jsonPayload, Collections.emptyMap());
+  }
+
+  @Override
+  public Mono<byte[]> invoke(String actorType,
+                             String actorId,
+                             String methodName,
+                             byte[] jsonPayload,
+                             Map<String, String> headers) {
```

Comment:
I assume the gRPC implementation is still in progress for this.

Reply:
Yes, the problem here is that the protobuf doesn't appear to expose Metadata to us, so I think this change might require a protobuf update.